configuration/service.go

```diff
@@ -31,13 +31,13 @@ const ServiceGrowthTypeExponential = "exponential"
 // ServiceServer contains information for one server in the service.
 type ServiceServer struct {
 	Address string
-	Port    int
+	Port    *int64
```

Comment:
Why Int64 ? TCP ports are just 16 bits.

Reply:
Simplifying types equality and assignment with other struct types.